edx_proctoring/templates/proctoring/seq_proctored_practice_exam_error.html

```diff
@@ -0,0 +1,56 @@
+{% load i18n %}
+<div class="failure sequence proctored-exam" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Your proctoring proctoring session is in error
+    {% endblocktrans %}
+  </h3>
+
+  <h4>
+    {% blocktrans %}
+      Your Practice Proctoring Session: <b class="failure"> Failed </b>
+    {% endblocktrans %}
+  </h4>
+
+  <p>
+    {% blocktrans %}
+      It appears that your proctoring session has been shut down while you were taking this
+      practice exam. Your practice exam is considered in error. While this will not affect your grade
+      or credit eligibility, it is recommended that you retry this practice exam until your
+      are successful
+    {% endblocktrans %}
+  </p>
+  <hr>
+  <div class="gated-sequence">
+    <span><i class="fa fa-lock"></i></span>
+    <a class="start-timed-exam" data-ajax-url="{{enter_exam_endpoint}}" data-exam-id="{{exam_id}}" data-attempt-proctored=true data-start-immediately=false>
+      {% trans "Please retry this practice exam again" %}
+    </a>
+  </div>
+</div>
+<script type="text/javascript">
+  $('.start-timed-exam').click(
+    function(event) {
+      var action_url = $(this).data('ajax-url');
+      var exam_id = $(this).data('exam-id');
+      var attempt_proctored = $(this).data('attempt-proctored');
+      var start_immediately = $(this).data('start-immediately');
+      if (typeof action_url === "undefined" ) {
+        return false;
+      }
+      $.post(
+        action_url,
+        {
+          "exam_id": exam_id,
+          "attempt_proctored": attempt_proctored,
+          "start_clock": start_immediately
+        },
+        function(data) {
+          // reload the page, because we've unlocked it
+          location.reload();
+        }
+      );
+    }
+  );
+</script>
```

Comment:
@chrisndodge Can we provide the same footer with the 3 "See Also"s as in the real exam templates?